pytorch_lightning/core/lightning.py

```diff
@@ -57,15 +57,14 @@
 log = logging.getLogger(__name__)
 
 
-class LightningModule(
+class _BaseLightningModule(
     ABC,
     DeviceDtypeModuleMixin,
```

Comment:
This one also inherits from Module, not sure if we have to have it here, but currently it does.

Also: if we say we don't want this one to inherit from nn.Module, we have to reimplement everything that is delegated back to nn.Module here again for the other lib